code/modules/economy/economy_machinery/atm.dm

```diff
@@ -208,17 +208,6 @@
 
 	. = TRUE
 
-/obj/machinery/economy/atm/proc/authenticate_account(account_number, account_pin, mob/user, silent = TRUE)
-	var/datum/money_account/target_account = account_database.find_user_account(account_number, include_departments = TRUE)
-	if(attempt_account_authentification(target_account, account_pin, user))
-		if(!silent)
-			to_chat(user, "[bicon(src)]<span class='notice'>Access granted. Welcome user '[authenticated_account.account_name].'</span>")
-		return TRUE
-	if(!silent)
-		playsound(src, 'sound/machines/buzz-two.ogg', 50, TRUE)
-		to_chat(user, "[bicon(src)]<span class='warning'>Authentification Failure, incorrect credentials or insufficient permissions.</span>")
-	return FALSE
-
```

Comment:
yeah I think this was leftover and it was replaced with the base economy machinery proc